Source/JavaScriptCore/jit/AssemblyHelpers.cpp

```diff
@@ -428,6 +429,17 @@ void AssemblyHelpers::emitLoadStructure(VM&, RegisterID source, RegisterID dest)
     emitNonNullDecodeZeroExtendedStructureID(dest, dest);
 }
 
+void AssemblyHelpers::emitEncodeStructure(RegisterID source, RegisterID dest)
+{
+#if ENABLE(STRUCTURE_ID_WITH_SHIFT)
```

Comment:
Have we tested with this configuration? Which platform is it?

Reply:
iOS

Reply:
It is iOS family.